runtime/jcl/common/jdk_internal_foreign_abi_UpcallStubs.cpp

```diff
@@ -102,7 +101,7 @@ Java_jdk_internal_foreign_abi_UpcallStubs_freeUpcallStub0(JNIEnv *env, jobject r
 			}
 		}
 	}
-	omrthread_monitor_exit(vm->thunkHeapWrapperMutex);
+	omrthread_monitor_exit(vm->thunkHeapListMutex);
 
 	return true;
```

Comment:
For `jboolean`, the appropriate value is `JNI_TRUE`.

Reply:
Agreed and fixed as suggested above.